docs/docs/concepts/low_level.md

```diff
@@ -322,6 +322,52 @@ def continue_to_jokes(state: OverallState):
 graph.add_conditional_edges("node_a", continue_to_jokes)
 ```
 
+## `GraphCommand`
```

Comment:
Could we somehow float this section to the top

```python
def my_node(state: State) -> GraphCommand[Literal["my_other_node"]]:
    return GraphCommand(update={"foo": "bar"}, goto="my_other_node")
```

These two lines is essentially everything a developer needs to know! Everything else is boiler plate